packages/perseus/src/types.ts

```diff
@@ -194,6 +194,12 @@ export type APIOptions = Readonly<{
      * answered correctly and should no longer be interactive.
      */
     readOnly?: boolean;
+    /**
+     * A boolean that indicates whether the editor interface should be
+     * disabled, preventing content creators from making changes — such
+     * as when we are performing Content Backfills or other migrations.
+     */
+    editingDisabled?: boolean;
```

Comment:
Thanks!  Last comment about this, but I think it'd be good to avoid writing documentation relating to internal/backend features of Khan Academy's systems. 
```suggestion
    /**
     * A boolean that indicates whether the editor interface should be
     * disabled, preventing content creators from making changes.
     */
    editingDisabled?: boolean;
```